### PR TITLE
Standardize source dir

### PR DESCRIPTION
**Pull request by jasonaden**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: packages/language-service/rollup.config.js [hunk 1]
```diff
@@ -10,21 +10,21 @@ import commonjs from 'rollup-plugin-commonjs';
...
+  'compiler-cli': normalize('../../dist/packages') + '/'
 };
 
 var esm_suffixes = {};
 
 function normalize(fileName) {
   return path.resolve(__dirname, fileName);
 }
-
+console.log('running')
```
Comment (IgorMinar) [inline packages/language-service/rollup.config.js]: poop

File: build.sh [hunk 2]
```diff
@@ -25,13 +25,18 @@ PACKAGES=(core
   language-service
   benchpress)
 
+NODE_TOOLS=(compiler-cli
```
Comment (IgorMinar) [inline build.sh]: consider renaming to `NODE_PACKAGES` to prevent confusion with stuff under `/tools`

File: build.sh [hunk 3]
```diff
@@ -50,41 +55,222 @@ for ARG in "$@"; do
...
 # Arguments:
-#   param1 - Destination folder
-#   param2 - Input path
-#   param3 - Output path
+#   param1 - Source folder
+#   param2 - Naming suffix to apply. Must end in ".ts" (defaults to .es5.ts)
+# Returns:
+#   None
+#######################################
+downlevelES2015_v2() {
```
Comment (IgorMinar) [inline build.sh]: remove `_v2`?

File: packages/core/tsconfig-build.json [hunk 4]
```diff
@@ -17,8 +17,7 @@
     "lib": ["es2015", "dom"],
     "skipLibCheck": true,
     // don't auto-discover @types/node, it results in a ///<reference in the .d.ts output
-    "types": [],
-    "preserveConstEnums": true
```
Comment (IgorMinar) [inline packages/core/tsconfig-build.json]: did you drop this option on purpose?

**Review by IgorMinar — CHANGES_REQUESTED** [review]
- `packages/examples/` and `packages/docs` are not "packages", we should find them a better home.

I still want to take a thorough look at the build.sh changes, but otherwise this looks pretty good!!!

File: packages/language-service/rollup.config.js (hunk 1 above)
Comment (jasonaden) [inline packages/language-service/rollup.config.js]: 👍

**Pull request merged**